### PR TITLE
Reverts atmospheric and computer icons.

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
imageadd: Switched back to original icons for computers, vents, and scrubbers.
/:cl:

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
fuck off and die

**Comment by Carbonhell** [comment]
+1

File: hippiestation.dme [hunk 1]
```diff
@@ -2374,7 +2374,7 @@
 #include "hippiestation\code\game\machinery\vending.dm"
 #include "hippiestation\code\game\machinery\vending_types.dm"
 #include "hippiestation\code\game\machinery\washing_machine.dm"
-#include "hippiestation\code\game\machinery\computer\arcade.dm"
```
Comment (Carbonhell) [inline hippiestation.dme:2377]: ??

**Comment by ghost** [comment]
we finally get good sprites merged and "HURRR DURRR I HATE CHANGE" comes in

**Comment by dreamcrab** [comment]
+888

**Comment by TrickyDickRichard** [comment]
good sprites LOL i love grey boxes with screens on them

**Comment by ghost** [comment]
I prefer being able to see a monitor & keyboard than a giant monitor with no visible form of input

**Comment by ghost** [comment]
How about we run a 7-day (DB) poll on this?

**Comment by TrickyDickRichard** [comment]
why the tg sprites are fuckin shit

**Comment by ghost** [comment]
@TrickyDickRichard No they aren't, you just hate any form of change

**Comment by ghost** [comment]
Or, even better, we implement a system where you can switch between the good /tg/ sprites and shit old sprites!

**Comment by dreamcrab** [comment]
how about no

**Comment by TrickyDickRichard** [comment]
you are so stupid amari stop telling your shitty opinion and go code something

**Comment by ghost** [comment]
@TrickyDickRichard Your opinion is also shit

**Comment by TrickyDickRichard** [comment]
didn't you brute force honkbot to add your change and break the server LOL stupid idiot get the fuck out

**Comment by ghost** [comment]
How about we do a poll rather than letting the world's most biased headmin decide

**Comment by ghost** [comment]
i signed just to post a +1

fucking LOL at these computer sprites

**Comment by dreamcrab** [comment]
why am i only biased when i disagree with your opinion

they look awful and are hard to identify from a distance, the other sprites are much clearer as to what they are

your complaint is that the old sprites dont have a keyboard and my complaint with the new sprites is gameplay related

**Comment by ghost** [comment]
@dreamcrab you just fucking hate change

**Comment by ghost** [comment]
also anyone who wasn't half blind could notice that they're computers

**Comment by dreamcrab** [comment]
its not that theyre just computers, nobody said they dont look like computers

what is on the screen is the important part. theyre all so similar looking whereas the screen on the old sprites were all very clear

**Comment by ghost** [comment]
untitled

**Comment by dreamcrab** [comment]
?????? what are you even arguing my man

File: hippiestation.dme (hunk 1 above)
Comment (Carbonhell) [reply hippiestation.dme:2377]: nvm im dumb

**Comment by ghost** [comment]
They're pretty much the same sprites, except with more detail and a different perspective

**Comment by TrickyDickRichard** [comment]
this is the old computer. it looks nothing alike to that fucking monstrosity
image

**Comment by ghost** [comment]
@TrickyDickRichard where's the computer pic? all i see is a dog turd

**Comment by TrickyDickRichard** [comment]
@MoreRobustThanYou do you like the new computer sprite because _you_ have parkinson's and the keyboard is made for people with it?

**Comment by ghost** [comment]
@TrickyDickRichard No, I like that it seems more realistic. Now go back to stroking your 50 nanometer dick.

**Comment by TrickyDickRichard** [comment]
how is it more realistic to have only 5 inch wide keys for your keyboard

**Comment by dreamcrab** [comment]
why do you know what his dick looks like dude

**Comment by fluxcapacitor1337** [comment]
You say company valuation is just a mere number but explain monopoly niche market? Samsung, LG, Hyundai, POSCO, these are all of our clients. Every year we rake in $150-170 million per year. The shiny gum wrapper chemical substance that you see on gum wrappers in Korea? $500 per week on groceries is nothing, I actually spend $2k a month on a maid. I don't go around bragging that **** but now I will. Yeah, we have the patent from Germany to make it SOLELY OURS and OURS only. The matte finish on 3M wraps? The ones produced in Korea, yeah, we made those in our factory. Company is worth MORE than $50 million. $50 million was the absolute minimum. The most I spent was $100k in 3 months span. I don't brag about that because I was young and dumb. Go ahead and school me, but you are just another ss13 player with nothing to prove except your posts and a faceless avi. And if you are, then fine congrats. But again read my previous response--I made github on this spacesttaion 13 server so people can ask me questions. Don't tell me you're going to "school" me when your head is filled with nothing but immaturity. You don't know me. Period.

**Comment by ghost** [comment]
@dreamcrab i don't, but that'd explain his attitude

**Comment by dreamcrab** [comment]
so you understand what it feels like to have a tiny penis is what you are saying

**Comment by ghost** [comment]
I do not care about the size of my dick

**Comment by ghost** [comment]
3 or 4 years ago I had to take the worlds biggest chit. I spent at least 45 minutes trying to get this massive turd out of my anus (serious). When all was said and done, I stood up and looked back at it and realized I had to see what it tasted like. So I got a knife and a fork (the turd was rock solid because I didn't drink much water back then) and I cut off a tiny piece, maybe 1/4th the size of someones pinky , and I tasted it. Honestly it didn't taste that bad, it was a crunchy kind of texture and it tasted kind of salty-ish, hard to describe.

**Comment by ghost** [comment]
Dick measuring is a barbarian concept

**Comment by TrickyDickRichard** [comment]
amari has autism and an small dick confirmed

**Comment by ghost** [comment]
+1

**Comment by Iamgoofball** [comment]
@JamieH you dipshit, 24 hour rule

**Comment by Blute1** [comment]
esketit get fucked TG

**Comment by ghost** [comment]
@Blute1 goofball made a good point, fuck off

**Comment by Blute1** [comment]
the TG sprites look like shit because we don't use their new topdown perspective (which totally isnt ripped off of goon)

**Comment by Blute1** [comment]
and by the way topdown perspective looks garbage anyways

**Comment by ghost** [comment]
@Blute1 did you know disliking /tg/ sprites is a sign of autism

**Comment by Sarcalogo** [comment]
Holy sh*t it's a coder fight!!!

**Comment by SplergeBucket** [comment]
@MoreRobustThanYou did you know being a coder is a sign of autism?
the old sprites fit in more with the theme
the new sprites are shit tg designs b/c maintaining a coherent art style is hard

**Comment by bardofrage** [comment]
+1

**Comment by Blute1** [comment]
it was already merged

**Comment by ghost** [comment]
Can you all stop being autistic

**Pull request merged**